packages/form-js-viewer/src/render/components/form-fields/Taglist.js

```diff
@@ -50,43 +49,43 @@ export default function Taglist(props) {
     values: options
   } = useValuesAsync(field);
 
+  // We cache a map of option values to their index so that we don't need to search the whole options array every time to correlate the label
+  const valueToOptionMap = useMemo(() => Object.assign({}, ...options.map((o, x) => ({ [o.value]:  options[x] }))), [ options ]);
```

Comment:
Maybe my eyes trick me because the expression is pretty long, but are we trying to assign an array (`.map` result) to an empty object?

Reply:
Yeah so this basically just turns ({a: x}, {b: y}) into {a: x, b: y}. It's just a short way to create an object from an array.

Reply:
There might be a better way to do this but I'm not certain.